swo_aws_extension/flows/jobs/billing_journal/item_journal_line.py

```diff
@@ -201,15 +199,21 @@ def _get_usage_journal_lines(
         journal_lines = []
         metric_dict = account_metrics.get(metric_id, {})
         skip_services = skip_services if skip_services else []
-        for sub_key, amount in metric_dict.items():
```

Comment:
If you don't mind for this function ->> blank lines in comments (before args and before Returns) :-)